starter-code/index.html

```diff
@@ -1,31 +1,50 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
   <title>Instagram</title>
+  <link rel="stylesheet" href="css/main.css">
 </head>
 <body>
-  Instagram
-
-  Capture and Share the World's Moments
-
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
-
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-
-  Oh yeah, did we mention it's free?
+    <div class="background">
 
-  Download on the App Store
-  Get it on Google Play
+    </div>
+    <div class="container">
+      <img src="images/phones.png" class="img-phones">
```

Comment:
I would put the phones inside a div. for example. <div class="section-left">image tag here</div>

css:
.section-left {
float: left;
}

Reply:
You could do something like this.

body {
  background: #fff url("../images/bgtile.png") repeat-x;
  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
  margin: 0;
}

// this is a <main> tag
main {
  width: 900px;
  margin: 90px auto 0;
}
// <section> with class section-left
main section.section-left {
  float: left;
}
// <section> with class section-right
main section.section-right {
  float: right;
  width: 500px;
}